bundle/edu.gemini.horizons.api/src/test/scala/edu/gemini/horizons/server/backend/HS2Spec.scala

```diff
@@ -31,11 +31,11 @@ object HS2Spec extends Specification with ScalaCheck {
 
     "handle multiple results" in {
       runSearch(Search.Comet("hu")).map(_.take(5)) must_== \/-(List(
-        Row(HD.Comet("67P"), "Churyumov-Gerasimenko"),
-        Row(HD.Comet("106P"), "Schuster"),
-        Row(HD.Comet("130P"), "McNaught-Hughes"),
-        Row(HD.Comet("178P"), "Hug-Bell"),
-        Row(HD.Comet("C/1880 Y1"), "Pechule")
+        Row(HD.Comet("7P"), "Churyumov-Gerasimenko"),
```

Comment:
I wonder why these are not stable ids. You may think they are primary/foreign keys somewhere

Reply:
This is a constant source of bafflement. There are apparently no keys.